front/Makefile

```diff
@@ -0,0 +1,202 @@
+DEV_IMAGE_NAME = dev_front
+
+#exec debug set to -i to see verbose or empty for no vebose
+LOG = -i
+# Installed application modules return 0 if exist 
+# In our case we want not exist before installing the app
+DEV_APP_INSTALLED = $(shell ls ./app/node_modules 2>&1 | grep -i "no such file" | wc -w)
+DEV_APP_INSTALLATION_LOCKER = $(shell ls ./app/package-lock.json 2>&1 | grep -i "no such file" | wc -w)
+APP_HOST_DIR = $(shell pwd)/app
+
+RED=\033[0;31m
+BG_RED=\033[0;41m
+GREEN=\033[0;32m
+ORANGE=\033[0;33m
+NC=\033[0m
+
+all:
+ifeq ($(DEV_APP_INSTALLATION_LOCKER), 0)
+	@echo "Starting the dev server"
+	@make -s -C . already_launch_serve
+	@make -s -C . start_container
+	@echo "$(ORANGE)Show help ? [y/n]$(NC)"
+	@read -r answer; \
+	if [ "$$answer" = "y" ] || [ "$$answer" = "Y" ]; then \
+		make -s -C . show_help_after_install; \
+	fi
+	@echo "$(GREEN)Happy coding inside your host folder !!!$(NC)"
+else
+	@echo "Please install the dev server with 'make install'"
```

Comment:
When I typed `make all` before `make install`, I thought I was going to trigger this condition, but I got this instead 
![Capture d’écran du 2023-07-24 16-29-57](https://github.com/AdrianWeaver/transcendence/assets/75798793/82d5f10c-fe7f-4111-aa9d-ecbd60086079)

Reply:
I need more information, what append in this situation if you do **make create_container** ? It seems you have a Folder that already have package-lock.json. Maybe **make fclean** to prevent this error until fix.

Reply:
I've just checked, you've got a previous installation of the app.
![image](https://github.com/AdrianWeaver/transcendence/assets/26166413/7e501d41-c50a-4301-b4e5-be6698a63e03)


Just remove package-lock.json and It will work fine(make clean)
![image](https://github.com/AdrianWeaver/transcendence/assets/26166413/15fdb5d5-234d-4c2b-be9e-ac673616fe34)

Let me know if you find another error :)